lib/apple_formatter.rb

```diff
@@ -42,7 +42,7 @@ def self.write_strings_file(terms, file, substitutions: nil, print_date: false,
 
         # If plural, use the text for the "one" (singular) entry
         if definition.is_a? Hash
-          definition = definition["one"]
+          definition = definition.values[0]
```

Comment:
Might be more complicated than that:
If we have a language that do support plurals, and have multiple keys, we still want to use whatever is considered like the definition to use for singular in the `.strings` file. But if definition is a Hash because we have plurals, the order of the key/value pair isn't guaranteed (as a hash as no concept of order)

So that change means that if we do have a hash with keys ordered like `{ "other": "…", "one": "…" }` that change will wrongly pick the plural translation to put in the `Localizable.strings` instead of the singular one

Reply:
So in practice we have to find a better test, like if it's a hash, then if it has a single key/value pair, then take the first (and only) value, if not then take the one for key `"one"` (and if there are more than one key but the key for `"one"` doesn't exist, then… look at what the official Unicode CLD says, of nothing, let it be nil or ignore it?)

Reply:
```suggestion
          definition = definition["one"] || definition.values.first
```

Reply:
Aha, really good catch!  😊👍

I liked your last suggestion, so I've made a commit with it. I can't see any risk with getting the wrong value, I mean if there's multiple values the first should always be "one", if I understood the plural rules correctly. I guess `.values.first` will return nil if it's a empty hash, which would give the "gSub" error, but that would at the same time enlighten that the translations are incorrect?

Reply:
Yep exactly.

1. `.values.first` indeed returns `nil` if `.values` is empty, which should never happen
2. But if that happens, which will lead to a similar crash stack trace that the one you observed with asian strings (nil.gsub will crash), it would mean something else went wrong, so enlighten that there would be something fishy ;-)

The only "risk" that I can see is if POEditor happens to return a hash with more than one value, but no `"one"` key in it, like `{"many": "…", "other": "…" }`, but I can't see in the plural rules and specs how that could happen, and what we'd be supposed to write in the `Localizable.strings` in such an unlikely case anyway, so…